### PR TITLE
add HP/Aruba JL693a and front image

### PR DESCRIPTION
**Pull request by wolfspyre**
Adds HPE/Aruba 2930F JL693a 12PoE+ / 2 NON-poe 2 10GSFP+ and front image.

I can also provide the raw SVG I made if it'd be helpful.

### CODE REVIEW

File: device-types/HPE/Aruba-2930F-12G-PoEP-2SFPP.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+manufacturer: HPE
+model: Aruba 2930F-12G-PoE+-2SFP+
+slug: hpe-aruba-2930f-12g-poep-2sfpp
+part_number: JL693A
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/HPE/Aruba-2930F-12G-PoEP-2SFPP.yaml:7]: Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Comment (danner26) [inline device-types/HPE/Aruba-2930F-12G-PoEP-2SFPP.yaml:7]: Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/HPE/Aruba-2930F-12G-PoEP-2SFPP.yaml (hunk 1 above)
Comment (wolfspyre) [reply device-types/HPE/Aruba-2930F-12G-PoEP-2SFPP.yaml:7]: updated

Comment (wolfspyre) [reply device-types/HPE/Aruba-2930F-12G-PoEP-2SFPP.yaml:7]: updated

**Comment by danner26** [comment]
Hello, we have released a new set of tests and features to assist with properly creating definition files for the DeviceType-Library. These tests will ensure that all files conform to a standardized format and contain valid data. Over the coming weeks we will be adding more tests to handle other currently undefined cases. If you notice anything that you believe is a bug, please @ me in a comment and details what you think is incorrect.

Along with additional test cases we have added a feature that will annotate why tests fail. To access these annotations after a test has completed, please navigate to the _Files changed_ tab within your PR. At the very bottom of the page you will see a list of errors along with the filename in question and the error which was found.

With all this being said, if your PR is validated then great! Please @ me me so I can review it further! If your PR is not valid, please review the issues and correct them; in the meantime I will mark your PR as a draft. After correcting the issues and recieving a valid status, please mark the PR as ready to be reviewed. Thank you!

**Comment by wolfspyre** [comment]
@danner26 - 
I'm sorry, I must be missing something? I thought this was good to go and pending y'all merging it... 

I'm happy to change something if you'd like, but I'm uncertain what change you're asking for (or if something's just gotten lost in translation)

If I understand your guidance correctly, running the `pre-commit run --all`  aught catch anything deviating from style/functional requirements.

Am I doing something wrong here?

```
(pyvenv-devicetypelibrary) ✔  16:28 /data/pyvenv-devicetypelibrary/devicetype-library [ HPE-2930F-JL693A {wplgh/HPE-2930F-JL693A} | ✔  ] # pip install pre-commit
Requirement already satisfied: pre-commit in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (3.3.3)
Requirement already satisfied: cfgv>=2.0.0 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from pre-commit) (3.3.1)
Requirement already satisfied: identify>=1.0.0 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from pre-commit) (2.5.24)
Requirement already satisfied: nodeenv>=0.11.1 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from pre-commit) (1.8.0)
Requirement already satisfied: pyyaml>=5.1 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from pre-commit) (6.0)
Requirement already satisfied: virtualenv>=20.10.0 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from pre-commit) (20.23.0)
Requirement already satisfied: setuptools in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from nodeenv>=0.11.1->pre-commit) (67.7.2)
Requirement already satisfied: distlib<1,>=0.3.6 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from virtualenv>=20.10.0->pre-commit) (0.3.6)
Requirement already satisfied: filelock<4,>=3.11 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from virtualenv>=20.10.0->pre-commit) (3.12.1)
Requirement already satisfied: platformdirs<4,>=3.2 in /data/pyvenv-devicetypelibrary/lib/python3.10/site-packages (from virtualenv>=20.10.0->pre-commit) (3.5.3)
(pyvenv-devicetypelibrary) ✔  16:29 /data/pyvenv-devicetypelibrary/devicetype-library [ HPE-2930F-JL693A {wplgh/HPE-2930F-JL693A} | ✔  ] # pre-commit run --all
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/jumanjihouse/pre-commit-hook-yamlfmt.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
Check YAML files.........................................................Passed
Format YAML files........................................................Passed
Lint YAML files..........................................................Passed
Run PyTest Case Checks...................................................Passed
(pyvenv-devicetypelibrary) ✔  16:30 /data/pyvenv-devicetypelibrary/devicetype-library [ HPE-2930F-JL693A {wplgh/HPE-2930F-JL693A} | ✔  ] #
(pyvenv-devicetypelibrary) ✔  19:22 /data/pyvenv-devicetypelibrary/devicetype-library [ HPE-2930F-JL693A {wplgh/HPE-2930F-JL693A} | ✔  ] # pre-commit run --all
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
Check YAML files.........................................................Passed
Format YAML files........................................................Passed
Lint YAML files..........................................................Passed
Run PyTest Case Checks...................................................Passed
(pyvenv-devicetypelibrary) ✔  23:25 /data/pyvenv-devicetypelibrary/devicetype-library [ HPE-2930F-JL693A {wplgh/HPE-2930F-JL693A} | ✔  ] # git status
On branch HPE-2930F-JL693A
Your branch is up to date with 'wplgh/HPE-2930F-JL693A'.

nothing to commit, working tree clean
(pyvenv-devicetypelibrary) ✔  23:33 /data/pyvenv-devicetypelibrary/devicetype-library [ HPE-2930F-JL693A {wplgh/HPE-2930F-JL693A} | ✔  ] # git push
Everything up-to-date
(pyvenv-devicetypelibrary) ✔  23:33 /data/pyvenv-devicetypelibrary/devicetype-library [ HPE-2930F-JL693A {wplgh/HPE-2930F-JL693A} | ✔  ] # git log --name-status
commit c90218826870ef09e36828c2de5e23f1baf00b02 (HEAD -> HPE-2930F-JL693A, wplgh/HPE-2930F-JL693A)
Merge: 35c3c0f 4ac4c79
Author: Daniel W. Anner <daniel.anner@danstechsupport.com>
Date:   Fri Jul 14 15:49:31 2023 -0400

    Merge branch 'master' into HPE-2930F-JL693A

commit 4ac4c79ab11cbd2a919d278b017d0652bef3fe5c
```

TIA!

🐺W

**Comment by danner26** [comment]
Hey W, sorry about the confusion here. My previous message was a generic blast sent to all PRs we pulled the master branch into yesterday. A majority of the open PRs validation status changed from passed to failed after the update, which was due to new model and/or file name validation tests. The purpose of the message was to let anyone know who was affected by this

Your PR did not fall into this category, so once more allow me to apologize for the confusion. As long as this PR is marked as ready for review and the checks are passing, I will get to reviewing the PR this coming week. Thank you for your patience!

**Review by danner26 — APPROVED** [review]

**Pull request merged**